en_US/rule/backend_hstreamdb.md

```diff
@@ -0,0 +1,98 @@
+# Data to HStreamDB
+
+HStreamDB is a streaming database designed for streaming data, with complete lifecycle management for accessing, storing, processing, and distributing large-scale real-time data streams. It uses standard SQL (and its stream extensions) as the primary interface language, with real-time as the main feature, and aims to simplify the operation and management of data streams and the development of real-time applications.
+
+![image](./assets/rule-engine/HStreamDB/HStream.png)
+
+For more details, please refer to [HStream official website](https://hstream.io/)。
+
+## Create HStreamDB Server
+
+[Deployment reference document](https://hstream.io/docs/en/latest/start/quickstart-with-docker.html). Can use docker local deployment, or cloud hosting deployment.
+Use the command to create hstream-client:
```

Comment:
Refer to the following [documents](https://hstream.io/docs/en/latest/start/quickstart-with-docker.html) for deployment instructions, which provide tutorials on deploying with docker locally or on the cloud.

Use the command to start an hstream-client